### DISCUSSION TITLE
Scene failed to load and selfies fail to load since February Hubs Cloud update

### DISCUSSION THREAD

**Discussion by wmurphyrd** [mozilla/hubs#3972] [Hubs Cloud Questions]
Starting with the February hubs cloud update, myself and at least one other discord have had some issues during local dev related the hostname being used in URLS. 

My setup - developing locally against my hubs cloud using `npm run start`

Issue 1: Scene failed to load ([discord discussion](https://discord.com/channels/498741086295031808/656208717813121034/807726548890550354)). The scene model and assets are being fetched not from `localhost` but instead from by pc's local network ip, e.g. `192.168.1.1`, on Chrome this request will be blocked with a certificate error unless you navigate directly to that ip and add another certificate exception

Issue 2: selfies fail to load. Internal domain (AWS recipe 2) is not recorgnized as a non CORS proxy domain and the local  ip is being used again, so I get a url like `https://192.168.1.1:8080/cors-proxy/https://hub-assets.internal-hub-domain.com/files/aaaa-bbbb.png?token=aaabbbbb` and that request gets rejected with `401 Unauthorized`. Note, if I edit the url in the html to remove the proxy and fetch directly from the internal domain, then the image loads

**Comment by markusTraber** [comment]
Can confirm the same behaviour. Although my server is on DO and currently working on a branch with qa-stage as base.
